sdcm/sct_config.py

```diff
@@ -1231,6 +1231,14 @@ class SCTConfiguration(dict):
         dict(name="perf_gradual_throttle_steps", env="SCT_PERF_GRADUAL_THROTTLE_STEPS", type=dict,
              help="Used for gradual performance test. Define throttle for load step in ops. Example: {'read': ['100000', '150000'], 'mixed': ['300']}"),
 
+        # StorageUtilizationTest
+        dict(name="auto_scaling_action_type", env="SCT_AUTO_SCALING_ACTION_TYPE", type=str,
+             help="Perform specific tasks (scale_in,scale_out etc)"),
```

Comment:
The description doesn't explain anything 

Also if there are multiple values supported, it should be listed as options=

Reply:
Updated it now.